src/redux/modules/reducer.js

```diff
@@ -13,5 +14,6 @@ export default combineReducers({
   clubs,
   hackbot,
   leaderIntake,
-  form
+  form,
+  techDomainRedemption
```

Comment:
The conventions are totally unclear for this file, which is on me. Can you put `techDomainRedemption` in alphabetical order of the modules that we've created? And then make sure that any modules we import from third-party libraries, like `form`, are at the bottom of `combineReducers`.